docs/devel/release-process.md

```diff
@@ -92,3 +99,59 @@ are issues that need to be handled before.
 If there is a security issue or an important bug fix we make patch releases in between. We don't
 have a formal definition of what an "important bug fix" is, it's up the team to discuss and decide
 whether to make a new release or not.
+
+## Making a bugfix release
+
+All newly merged patches fixing previous bug have to be backported to the latest stable release.
+When you merge a PR containing a patch fixing a bug, you should test if the buggy commits belongs to the latest release.
+Let's take an example with this commit:
+
+```patch
+commit 95fe7405738d58476ddad29856ebe30599644666
+Author: author <author@mail.com>
+Date:   Tue Jan 9 16:42:38 2024 +0100
+
+    check for empty record in capabilities tracer to avoid a panic
+
+    Fixes: 39aefb92dd1df2ff73647b17707984662f8718c0 ("pkg/gadgets: Add capabilities CO-RE tracer.")
+    Signed-off-by: author <author@mail.com>
+```
+
+To test if the buggy commit belong to the stable release, you can use the following:
+
+```bash
+$ git tag --contains 39aefb92dd1df2ff73647b17707984662f8718c0 --sort=-v:refname
+v0.x.y
+...
+v0.x.1
+v0.x.0
+v0.24.0
+v0.23.1
+```
+
+You now need to backport the patch to the latest stable branch:
+
+```bash
+$ git checkout v0.x-stable
+$ git checkout -b v0.x-stable/fix-something
+$ git cherry-pick 95fe7405738d58476ddad29856ebe30599644666
+# If the patch does not apply cleanly, you will need to adapt it.
+# Once done, you can just push your branch and open a PR:
+$ git push --set-upstream origin v0.x-stable/fix-something
+```
+
+Even if the patch applies cleanly, you need to open a PR for these two main reasons:
+
+1. It will run the integration test, ensuring backporting the patch does not create bug.
+2. You may get reviews from another reviewer who could catch different problems.
+
+Once your PR is merged, you can now create the bugfix tag:
+
+```bash
+$ git checkout v0.x-stable
+$ git pull
+# Where z would be y + 1:
+$ git tag v0.x.z
+```
+
+Once done, follow the instructions listed [here](/docs/devel/release-process.md#making-a-new-release), to create a new release.
```

Comment:
Can't you skip the name of the document and only  using the anchor for the link?